alexandria/core/models.py

```diff
@@ -119,6 +121,16 @@ class Document(UUIDModel):
 
 
 class File(UUIDModel):
+    ORIGINAL = "original"
+    THUMBNAIL = "thumbnail"
+
+    TYPE_CHOICES = (
+        ORIGINAL,
+        THUMBNAIL,
+    )
+    TYPE_CHOICES_TUPLE = ((type_choice, type_choice) for type_choice in TYPE_CHOICES)
+    type = models.CharField(choices=TYPE_CHOICES_TUPLE, max_length=23, default=ORIGINAL)
+    family = models.ForeignKey("self", on_delete=models.CASCADE, null=True)
```

Comment:
Wouldn't it be cleaner to have `family` named `original` and set `related_name="thumbnails"`? Or what is the family going to be used for?

Reply:
I'd agree, I prefer `family` to the case where we're dealing with trees.

Reply:
Done as discussed.